builder/comp-builder.nix

```diff
@@ -340,7 +350,8 @@ let
     };
 
     propagatedBuildInputs =
-         frameworks # Frameworks will be needed at link time
+      # Frameworks will be needed at link time
+      lib.optionals stdenv.hostPlatform.isDarwin frameworks
```

Comment:
Shouldn't the option be empty unless we're on darwin anyway?

Reply:
`plan-to-nix` populates it anyway.  Sometimes it is hidden behind an `if os(darwin)` in the `.cabal` file, but I am not sure if that should be required.

Reply:
Okay, https://cabal.readthedocs.io/en/latest/cabal-package.html#pkg-field-frameworks explicitly says it should be ignored on all other platforms so that seems legit (maybe worth linking to the doc).